__tests__/integration.test.js

```diff
@@ -218,3 +218,46 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  it("PATCH:200 should respond with the updated article when successful", () => {
+    const newVote = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/9")
+      .send(newVote)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(1);
+        expect(typeof article.article_id).toBe("number");
```

Comment:
You know the article_id, so I think the expectation should not be about the type :)

Reply:
sorted